app/Trade/Trade.php

```diff
@@ -30,6 +30,17 @@ class Trade extends Model
    */
   protected $hidden = [];
 
+  public function getSummaryAttribute() {
+    $c = [];
+
+    foreach ($this->exchanges as $e) {
+      $c[$e->gainingTeam->id]['gains'][] = $e->asset->id;
```

Comment:
Note to self for morning: Add another level of nesting. Asset is polymorphic so should the output be.

```
trade: {
  teamA: {
    gains: {
      players: [1, 2, 3],
      picks: [15, 275, 2],
      naughtyFavours: [99, 999, 9999]
    }
  }
}
```

@quagliero agreed?